lib/plaid/connection.rb

```diff
@@ -9,22 +9,29 @@ class << self
       def post(path, options = {})
         uri = build_uri(path)
         options.merge!(client_id: Plaid.customer_id, secret: Plaid.secret)
-        res = Net::HTTP.post_form(uri,options)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = (uri.scheme == 'https')
```

Comment:
i think Plaid always uses SSL, so this `http.use_ssl` should be just strictly `true`